tasks/tool.py

```diff
@@ -2,6 +2,7 @@
 import json
 import pathlib
 import platform
+import subprocess
```

Comment:
<!-- Suggestions propose improvements to the current subject. It’s important to be explicit and clear on what is being suggested and why it is an improvement. -->
**💬 suggestion**
You should be able to use [invoke context run](https://docs.pyinvoke.org/en/stable/api/runners.html) to execute commands, we use it to call `pulumi` in [deploy](https://github.com/DataDog/test-infra-definitions/blob/main/tasks/deploy.py#L132)

Reply:
Done!